### ISSUE TITLE
About data preprocessing

### ISSUE THREAD

**Issue by 408550969** [ubicomplab/rPPG-Toolbox#280]
Hello, I am not familiar with RPPG. I would like to confirm if my data preprocessing is correct. Firstly, I collected PPG signals. For example, I collected 10 videos, each lasting 1 minute. Then I divided the video into images and matched the PPG signal with each image. Then I normalized the PPG signal of each video segment with a mean of 0 and a standard deviation of 1. Finally, conduct training. What I want to ask is whether the normalization operation is performed separately for each video segment (i.e. 10 videos are normalized 10 times each), or for all videos (i.e. all training sets are normalized only once)?

**Comment by 408550969** [comment]
Or does the dataset not need to be normalized at all? When I saw the ground_truth file of UBFC and saw e-01 in front of it, I instinctively thought that normalization was necessary. However, I saw a value of e+02 at the end. Is it possible for datasets like UBFC not to perform operations with a mean of 0 and a standard deviation of 1, but to directly train the read ppg values?
